### PR TITLE
1247 nrwl 13

### LINKED ISSUE #1247: Upgrade to nrwl 13
Since webpack less compile has issue with nextjs in nrwl 13, which uses webpack 5. we have compiled the less to CSS file & import directly.

### PR DESCRIPTION
**Pull request by phmngocnghia**
## Expected Behavior
- Upgrade nwrl to 13
- Enable webpack 5 and fix LESS import

## Related Issue(s)

Fixes #1247

### CODE REVIEW

File: apps/web/pages/_app.tsx [hunk 1]
```diff
@@ -1,7 +1,10 @@
 // import '../src/wdyr'
-import '../src/styles/antd-theme.less'
-import 'animate.css'
-import '../src/styles/app.scss'
+// Comment out in favor of compiled css file
```
Comment (phmngocnghia) [inline apps/web/pages/_app.tsx:2]: still testing this thing

**Comment by phmngocnghia** [comment]
Status: Investigating some issues with memory leak, infinity loop when upgrading to webpack 5

File: libs/backend/infra/src/modules/dgraph/config/dgraph.config.ts [hunk 2]
```diff
@@ -10,7 +10,8 @@ export interface DgraphConfig {
 export const dgraphConfig = registerAs<DgraphConfig>(
   DgraphTokens.DgraphConfig.toString(),
   () => ({
-    endpoint: get('CODELAB_DGRAPH_ENDPOINT').required().asUrlString(),
+    // We set DG_AUTH inside CircleCI config with CODELAB_DGRAPH_INTEGRATION_ENDPOINT or CODELAB_DGRAPH_INTEGRATION_DG_AUTH
+    endpoint: '',
```
Comment (webberwang) [inline libs/backend/infra/src/modules/dgraph/config/dgraph.config.ts:14]: This actually causes an error, could rebase to revert

Comment (phmngocnghia) [reply libs/backend/infra/src/modules/dgraph/config/dgraph.config.ts]: fixed

**Comment by webberwang** [comment]
Closed by #1263

**Pull request closed without merging**